js/main.js

```diff
@@ -0,0 +1,26 @@
+// Функция взята из источника https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+const randomInt = function getRandomIntInclusive(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  if (min >= 0 && max >= 0) {
+    if (min < max) {
+      return Math.floor(Math.random() * (max - min + 1)) + min;
+    }
+    return alert('Значение «до» меньше, чем значение «от», или равное ему');
```

Comment:
Можно бросать ошибку.
```suggestion
    throw new Error('Значение «до» меньше, чем значение «от», или равное ему');
```